packages/gatsby-theme-docs/src/layouts/internals/page-course-status.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useCourseStatusByCouseId } from '@commercetools-docs/gatsby-theme-learning';
+import PropTypes from 'prop-types';
+
+const CourseStatus = (props) => {
+  if (props.error) {
+    return <span>unavailable</span>;
```

Comment:
From the screenshots that Gabriele sent, I'd agree to go with the first approach and not show anything to a user.

_(Please note that if this was a critical component for a user to progress, I'd suggest showing that there's an error displayed so that users get feedback re what's happening.)_